lib/logux/base_controller.rb

```diff
@@ -19,6 +19,10 @@ def initialize(action:, meta: {})
       @meta = meta
     end
 
+    def send_back(action, meta = @meta)
```

Comment:
We do not need to copy origin `@meta`.

`BaseController` is a controller to process action (an object with a description of what to change in a database). Each action has meta (with unique ID and time when it was created).

`@action` and `@meta` here is an action from the client, which we should process.

`action` and `meta` in `send_back()` is an action, which server answer to client’s action.

By setting `meta = @meta` we will force our server’s response action to have the same ID, that the client’s action had. Because the server’s action will have the same ID, the client will ignore this new server’s action, because of client already have action with the same ID.

Reply:
```suggestion
    def send_back(action, meta = { })
```

Reply:
Ok, got it now. So, I just take `@meta`'s client ID and add it to passed meta, to say Logux, that I want to send the action back to the client.